decidim-proposals/app/views/decidim/proposals/proposals/show.html.erb

```diff
@@ -50,7 +50,7 @@ extra_admin_link(
         <%= cell "decidim/coauthorships", @proposal, context_actions: [:date] %>
 
         <% if not ["section","subsection"].include? @proposal.participatory_text_level %>
-          <%== cell("decidim/proposals/proposal_metadata", @proposal).state_item&.dig(:text)&.html_safe %>
+          <%= cell("decidim/proposals/proposal_metadata", @proposal).state_item&.dig(:text) %>
```

Comment:
I could not find any difference in the UI after changing this, but it wouldn't hurt a second pair of eyes on this one

Reply:
I have not checked this, but as far as i can see, the output of the state_item is already a content_tag erb element.